osu.Game/Screens/Backgrounds/BackgroundModeBeatmap.cs

```diff
@@ -22,17 +22,19 @@ public WorkingBeatmap Beatmap
             {
                 return beatmap;
             }
-
             set
             {
-                if (beatmap == value)
+                if (beatmap == value && beatmap != null)
                     return;
-
                 beatmap = value;
 
                 Schedule(() =>
                 {
-                    Background newBackground = new BeatmapBackground(beatmap);
+                    Background newBackground;
+                    if (beatmap == null)

```

Comment:
But only on this screen. You can't get to the player if there's no song for you to select here. I'm not suggesting we allow every screen to have a null beatmap, it makes sense of course for Player to require a non-null beatmap for example.